samples/Core-Bot/dialogs/booking_dialog.py

```diff
@@ -4,26 +4,33 @@
 from .cancel_and_help_dialog import CancelAndHelpDialog
 from .date_resolver_dialog import DateResolverDialog
 
+from datatypes_timex_expression.timex import Timex
+
 class BookingDialog(CancelAndHelpDialog):
 
     def __init__(self, dialog_id: str = None):
         super(BookingDialog, self).__init__(dialog_id or BookingDialog.__name__)
 
         self.add_dialog(TextPrompt(TextPrompt.__name__))
-        self.add_dialog(ConfirmPrompt(ConfirmPrompt.__name__))
+        #self.add_dialog(ConfirmPrompt(ConfirmPrompt.__name__))
         self.add_dialog(DateResolverDialog(DateResolverDialog.__name__))
         self.add_dialog(WaterfallDialog(WaterfallDialog.__name__, [
-
+            self.destination_step,
+            self.origin_step,
+            self.travel_date_step,
+            #self.confirm_step,
+            self.final_step
         ]))
 
-        self.initial_dialog_id(WaterfallDialog.__name__)
+        self.initial_dialog_id = WaterfallDialog.__name__
     
     """
     If a destination city has not been provided, prompt for one.
     """
```

Comment:
Complete nit: We were attempting this style of docstring:
[https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)
But then autorest appears to be adopting reStructuredText, so we started following that to be consistent..